service/core/client/src/main/resources/reference.conf

```diff
@@ -21,24 +21,24 @@ lagom.circuit-breaker {
     # the circuit, by first entering the half-open state.
     reset-timeout = 15s
 
-    # A whitelist of fqcn of Exceptions that the CircuitBreaker
+    # A allowed exceptions of fqcn that the CircuitBreaker
     # should not consider failures. By default all exceptions are
     # considered failures.
-    exception-whitelist = []
+    allowed-exceptions = []
```

Comment:
This is problematic because it breaks backwards compatibility. 

I agree that `allowed` is much clear then `whitelist`, but this affects user code directly. 

Also, the code that reads that setting wasn't changed and at runtime it will try to read the `exception-whitelist` 

So better to revert that part.

Reply:
> This is problematic because it breaks backwards compatibility.
> 
> I agree that `allowed` is much clear then `whitelist`, but this affects user code directly.
> 
> Also, the code that reads that setting wasn't changed and at runtime it will try to read the `exception-whitelist`
> 
> So better to revert that part.

@renatocaval I have changed that code that read this setting

Reply:
@renatocaval Formatting problem is coming in Dependencies.scala

Reply:
That's still backwards-incompatible change. 

Users are setting `lagom.circuit-breaker.exception-whitelist` on their configuration files. 

With this change, `CircuitBreakersPanelInternal` is now reading from `lagom.circuit-breaker.allowed-exception` and therefore ignoring user-defined settings. 

We can only change that with a deprecation warning. You need to read the old and new settings. And emit a warning if the old is existing and non-empty. Then later we need to remove it and add to the migration guide.

That's why I suggested reverting that part. This is too much work for little value. 

Note that this is also unrelated to the changes in the build. The only similarity is the usage of the word 'whitelist'. I would prefer that we don't change that. And if we really find it's necessary, that we do it on another PR and include the necessary deprecation warnings.